.jenkins/build.groovy

```diff
@@ -38,7 +38,7 @@ pipeline {
                     stage('Build') {
                         steps {
                             sh '''
-                                make build
+                                make venv install build
```

Comment:
`build` target should have a dependency on `install`.
`install` should have on `create_venv`

Reply:
bea0ae1acc67f38ecd23226b2cd47310d0654329
517b50622c25892bc5db7ab0c24da2a8114d8e9e